scripts/enrich_quick_note_issues.py

```
@@ -0,0 +1,110 @@
+    return h
+
+
+def _fetch_open_issues() -> list[dict]:
+    res = requests.get(f"{API}/issues", params={"state": "open", "per_page": 100}, headers=_headers(), timeout=30)
+    res.raise_for_status()
+    return [i for i in res.json() if "pull_request" not in i]
+
+
+def _is_quick_note(issue: dict) -> bool:
+    title = issue.get("title", "").lower()
+    labels = [l["name"].lower() for l in issue.get("labels", [])]
```

Ruff (0.15.13)

[error] Ambiguous variable name: `l`

(E741)